anchore_engine/services/apiext/api/helpers/image_content_response.py

```diff
@@ -0,0 +1,220 @@
+    response = []
+    for package_name, package_info in content_data.items():
+        el = {}
+        try:
+            el["package"] = package_name
+            for field in ["license", "origin", "size", "type", "version"]:
+                if field in package_info:
+                    el[field] = package_info[field]
+                else:
+                    el[field] = None
+
+                if field == "license":
+                    if el[field]:
+                        el["licenses"] = el[field].split(" ")
+                    else:
+                        el["licenses"] = []
+
+            # Special formatting for rpms
+            if package_info.get("type", "").lower() in ["rpm"]:
+                v = package_info.get("version", None)
+                r = package_info.get("release", None)
+                if (v and r) and (v.lower() != "n/a") and r.lower() != "n/a":
+                    el["version"] = "{}-{}".format(v, r)
+        except:
+            continue
+        response.append(el)
+    return response
+
+
+def _build_npm_response(content_data):
+    response = []
+    for package in list(content_data.keys()):
+        el = {}
+        try:
+            el["package"] = content_data[package]["name"]
+            el["type"] = "NPM"
+            el["location"] = package
+            el["version"] = content_data[package]["versions"][0]
+            el["origin"] = ",".join(content_data[package]["origins"]) or "Unknown"
+            el["license"] = " ".join(content_data[package]["lics"]) or "Unknown"
+            el["licenses"] = content_data[package]["lics"] or ["Unknown"]
+        except:
+            continue
+        response.append(el)
+    return response
+
+
+def _build_gem_response(content_data):
+    response = []
+    for package in list(content_data.keys()):
+        el = {}
+        try:
+            el["package"] = content_data[package]["name"]
+            el["type"] = "GEM"
+            el["location"] = package
+            el["version"] = content_data[package]["versions"][0]
+            el["origin"] = ",".join(content_data[package]["origins"]) or "Unknown"
+            el["license"] = " ".join(content_data[package]["lics"]) or "Unknown"
+            el["licenses"] = content_data[package]["lics"] or ["Unknown"]
+        except:
+            continue
+        response.append(el)
+    return response
+
+
+def _build_python_response(content_data):
+    response = []
+    for package in list(content_data.keys()):
+        el = {}
+        try:
+            el["package"] = content_data[package]["name"]
+            el["type"] = "PYTHON"
+            el["location"] = content_data[package]["location"]
+            el["version"] = content_data[package]["version"]
+            el["origin"] = content_data[package]["origin"] or "Unknown"
+            el["license"] = content_data[package]["license"] or "Unknown"
+            el["licenses"] = content_data[package]["license"].split(" ") or ["Unknown"]
```

Comment:
Why does it need to default to a list with `["Unknown"]` instead of an empty list?

Reply:
Good point, I was trying to keep to the same behavior as before, which was not to list an empty string when you could not determine the presence of licenses, but to explicitly list that the license is "Unknown"... I kept suite with the same logic for multiple licenses (that is, there aren't explicitly 0 licenses, instead the licenses could not be positively determined).

I'm not 100% in love with `["Unknown"]` but am open for suggestions.